lib/pidfile/lib/pidfile.c

```diff
@@ -19,9 +19,10 @@
 #include <bsd/string.h>
 #include <bsd/libutil.h>
 
+#include <error/merr.h>
 #include <pidfile/pidfile.h>
 
-int
+merr_t
```

Comment:
Why did you change this from int?

Reply:
That was a mistake from me about not updating the returns. Checking it out now.